src/frontend/context/SpråkContext.tsx

```diff
@@ -9,17 +8,22 @@ import { erGyldigSpråk, LocaleType } from '../typer/common';
 
 const dekoratorLanguageCookieName = 'decorator-language';
 
-export const [SpråkProvider, useSpråk] = createUseContext(() => {
+interface ISpråkContext {
+    valgtLocale: LocaleType;
+}
```

Comment:
Jeg tenker vi kan like greit droppe `I` prefixen da typescript har typer. 😅 Ser det er gjort mange andre plasser også, så sikkert derfor du har gjort det her, men tenker det er helt greit å snu den trenden. Tanker? Blir jo en del jobb hvis man bestemmer seg for å f.eks. gjøre det om til en `type` senere 

```suggestion
interface SpråkContext {
    valgtLocale: LocaleType;
}
```

Reply:
I fix 🙌